src/Sylius/Bundle/UiBundle/Resources/private/sass/main.scss

```diff
@@ -110,3 +110,31 @@ th {
   right: 0;
   left: auto;
 }
+div.coupon.field {
+  width: 75% !important;
+}
+
+@media only screen and (max-width: 1200px) {
```

Comment:
I would put all `@media` tags together. Moreover, body of this block can be placed in the same block below `@media only screen and (max-width: 1200px)`.